src/Microsoft.AspNet.Server.Kestrel/Http/FrameHeaders.Generated.cs

```diff
@@ -5623,159 +5422,13 @@ public StringValues HeaderWWWAuthenticate
-                {
-                    ++count;
-                }
-            
-                if (((_bits & 2048L) != 0)) 
-                {
-                    ++count;
-                }
-            
-                if (((_bits & 4096L) != 0)) 
-                {
-                    ++count;
-                }
-            
-                if (((_bits & 8192L) != 0)) 
-                {
-                    ++count;
-                }
-            
-                if (((_bits & 16384L) != 0)) 
-                {
-                    ++count;
-                }
-            
-                if (((_bits & 32768L) != 0)) 
-                {
-                    ++count;
-                }
-            
-                if (((_bits & 65536L) != 0)) 
-                {
-                    ++count;
-                }
-            
-                if (((_bits & 131072L) != 0)) 
-                {
-                    ++count;
-                }
-            
-                if (((_bits & 262144L) != 0)) 
-                {
-                    ++count;
-                }
-            
-                if (((_bits & 524288L) != 0)) 
-                {
-                    ++count;
-                }
-            
-                if (((_bits & 1048576L) != 0)) 
-                {
-                    ++count;
-                }
-            
-                if (((_bits & 2097152L) != 0)) 
-                {
-                    ++count;
-                }
-            
-                if (((_bits & 4194304L) != 0)) 
-                {
-                    ++count;
-                }
-            
-                if (((_bits & 8388608L) != 0)) 
-                {
-                    ++count;
-                }
-            
-                if (((_bits & 16777216L) != 0)) 
-                {
-                    ++count;
-                }
-            
-                if (((_bits & 33554432L) != 0)) 
-                {
-                    ++count;
-                }
-            
-                if (((_bits & 67108864L) != 0)) 
-                {
-                    ++count;
-                }
-            
-                if (((_bits & 134217728L) != 0)) 
-                {
-                    ++count;
-                }
-            
-                if (((_bits & 268435456L) != 0)) 
-                {
-                    ++count;
-                }
-            
-                if (((_bits & 536870912L) != 0)) 
-                {
-                    ++count;
-                }
-            
-            return count;
+            // Parallel bit count for a 64-bit integer
+            var v = (ulong)_bits;
+            v = v - ((v >> 1) & 0x5555555555555555);
+            v = (v & 0x3333333333333333) + ((v >> 2) & 0x3333333333333333);
+            v = (v + (v >> 4) & 0x0f0f0f0f0f0f0f0f);
+            var count = (int)((v * 0x0101010101010101) >> 56);
+            return count + (MaybeUnknown?.Count ?? 0);
```

Comment:
IIRC `MaybeUnknown?.Count` has to hoist `.Count` into a `Nullable<int>`. Profiler would know for sure